src/lib/seed-db.js

```diff
@@ -29,13 +30,13 @@ function * setupAdminAccount (adminParams) {
     adminAccount.fingerprint = adminParams.fingerprint
     yield upsertAccount(adminAccount)
   } else {
-    yield upsertAccount({
+    yield upsertAccount(converters.convertToInternalAccount({
```

Comment:
Wouldn't it be easier just to pass an internal account, since its being constructed here anyway? Whichever route you go, the `upsertAccount()` call in `setupHoldAccount()` needs the same treatment.